### PR TITLE
Add Cards Fetch endpoint, Increment version to v3

### PR DESCRIPTION
**Pull request by jbabichjapan**
1. Add cards fetch endpoint (and example)
2. Increment version to v3

### CODE REVIEW

File: lib/twitter-ads/client.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 module TwitterAds
 
-  API_VERSION = '2'.freeze
+  API_VERSION = '3'.freeze
```
Comment (tushdante) [inline lib/twitter-ads/client.rb]: this should be unnecessary given that i've made this change in a [previous PR](https://github.com/twitterdev/twitter-ruby-ads-sdk/pull/171/files#diff-159cb692d402ced7dfc6394a9a255caaR5)

File: lib/twitter-ads/creative/cards_fetch.rb [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      property :website_url, read_only: true
+      property :wide_app_image, read_only: true
+      property :id, read_only: true
+
+      FETCH_URI = "/#{TwitterAds::API_VERSION}/" +
+                  'accounts/%{account_id}/cards'.freeze # @api private
+      FETCH_ID  = "/#{TwitterAds::API_VERSION}/" +
+                  'accounts/%{account_id}/cards/all/%{id}'.freeze # @api private
+
+      def all(*)
```
Comment (tushdante) [inline lib/twitter-ads/creative/cards_fetch.rb]: is the `*` necessary here? should be able to just write this as `def all`

File: lib/twitter-ads/creative/cards_fetch.rb [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        raise ArgumentError.new(
+          "'CardsFetch' object has no attribute 'all'")
+      end
+
+      def load(account, card_uri = nil, card_id = nil, with_deleted = nil, opts = {})
+        if card_uri && card_id
```
Comment (tushdante) [inline lib/twitter-ads/creative/cards_fetch.rb]: does this check work with the scenario where neither are defined?

File: lib/twitter-ads/client.rb (hunk 1 above)
Comment (jbabichjapan) [reply lib/twitter-ads/client.rb]: This is necessary as discussed offline

File: lib/twitter-ads/creative/cards_fetch.rb (hunk 2 above)
Comment (jbabichjapan) [reply lib/twitter-ads/creative/cards_fetch.rb]: This is weird looking but just a catch all (and was straight from rubocop...)

File: lib/twitter-ads/creative/cards_fetch.rb (hunk 3 above)
Comment (jbabichjapan) [reply lib/twitter-ads/creative/cards_fetch.rb]: I am updating this one right now

Comment (jbabichjapan) [reply lib/twitter-ads/creative/cards_fetch.rb]: Updated

File: lib/twitter-ads/creative/cards_fetch.rb [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+      property :wide_app_image, read_only: true
+      property :id, read_only: true
+
+      FETCH_URI = "/#{TwitterAds::API_VERSION}/" +
+                  'accounts/%{account_id}/cards'.freeze # @api private
+      FETCH_ID  = "/#{TwitterAds::API_VERSION}/" +
+                  'accounts/%{account_id}/cards/all/%{id}'.freeze # @api private
+
+      def all(*)
+        raise ArgumentError.new(
```
Comment (juanshishido) [inline lib/twitter-ads/creative/cards_fetch.rb]: What do you think about raising `NoMethodError` instead? With some string like:

>"NoMethodError: undefined method `all' for CardsFetch"

[Source](http://ruby-doc.org/core-1.9.3/NoMethodError.html)

**Comment by amartinezg** [comment]
@jbabichjapan @juanshishido According to what a I see in other classes, shouldn't this `load` method be a class method? Right now is an instance method and I have to do something like this to get the cards associated to one account:

`TwitterAds::Creative::CardsFetch.new(account).load(account, card_uri)`

**Comment by juanshishido** [comment]
Thanks for the feedback, @amartinezg. We appreciate the community taking a close look at changes!

It sounds like what you're suggesting is that the interface should be:

```
TwitterAds::Creative::CardsFetch.load(account, card_uri)
```

That makes sense.

If you're able to submit a pull request, we'll prioritize reviewing it. If not, we'll add this to our backlog.

Thanks again!

**Pull request merged**